src/blocks/index.js

```diff
@@ -4,3 +4,4 @@ export const ArticleSlideshow = dynamic(() => import(/* webpackChunkName: "block
 export const Content = dynamic(() => import(/* webpackChunkName: "blocks/Content" */ './Content')) // prettier-ignore
 export const Hero = dynamic(() => import(/* webpackChunkName: "blocks/Hero" */ './Hero')) // prettier-ignore
 export const Media = dynamic(() => import(/* webpackChunkName: "blocks/Media" */ './Media')) // prettier-ignore
+export const Promo = dynamic(() => import(/* webpackChunkName: "blocks/Promo" */ './Media')) // prettier-ignore
```

Comment:
Excellent work on exporting the block file. A lot of n00bs keep forgetting this :)